 .coderabbit.yaml

```
@@ -16,6 +16,11 @@ reviews:
   enabled: true
   review_status: true
   profile: "chill"
+  auto_review:
+    enabled: true
+    drafts: false
   branches:
     include:
       - "*"
+  chat:
+    auto_reply: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)